docs/config/supernode_properties.md

```diff
@@ -0,0 +1,72 @@
+# Customize supernode properties
+
+This topic explains how to customize the dragonfly supernode startup parameters.
+
+## Parameter instructions
+
+### The parameters we can configure in supernode is as follows
+
+The following startup parameters are supported for `supernode`
+
+| Parameter | Default | Description |
+| ------------- | ------------- | ------------- |
+| listenPort | 8002 | listenPort is the port that supernode server listens on |
+| downloadPort | 8001 | downloadPort is the port for download files from supernode |
+| homeDir | /home/admin/supernode | homeDir is the working directory of supernode |
+| advertiseIP | localhost | the supernode ip is the ip we advertise to other peers in the p2p-network |
+| schedulerCorePoolSize | 10 | pool size is the core pool size of ScheduledExecutorService |
```

Comment:
In fact, `schedulerCorePoolSize` is unused for now. And make a record here.

Reply:
done